packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -1709,15 +1750,14 @@ describe("renderer", () => {
             act(() => jest.runOnlyPendingTimers());
 
             // Act
-            const userInput = renderer.getUserInputForWidgets();
+            const userInput = renderer.getUserInputMap();
 
             // Assert
             expect(userInput).toMatchInlineSnapshot(`
                 {
                   "dropdown 1": {
                     "value": 1,
                   },
-                  "image 1": null,
```

Comment:
I'm hoping this isn't a problem either. I don't see the point in storing "user data" for widgets that aren't interactive.

Reply:
Is there anywhere that userInput needs to reflect all the widgets that are available? I could see it possibly being useful for that, but it does seem silly for userInput to track widgets that will never have userInput.